### DISCUSSION TITLE
Generating types from schema

### DISCUSSION THREAD

**Discussion by adamalfredsson** [tinacms/tinacms#4350] [Help] [answered]
Hi!

The documentation [here](https://tina.io/docs/cli-overview/#tinacms-dev) states that typescript types will be generated when running `tinacms dev`, but I'm unable to find any in the `__generated__` directory. I'm not using GraphQL, so I'm looking for type inference from the schema.

Thanks!

**Comment by kldavis4** [comment]
They should be in `tina/__generated__/types.ts`

**Comment by adamalfredsson** [comment, marked as answer]
Ah, turned out I had set:

```typescript
{
  client: { skip: true }
}
```

In tinacms config
